primitives/tf_encrypted/primitives/sodium/cc/easy_box_kernels.cc

```diff
@@ -159,15 +162,16 @@ class SodiumEasyBoxOpenDetached : public OpKernel {
 
     Tensor* plaintext_t;
     TensorShape plaintext_shape = ciphertext_t.shape();
+    plaintext_shape.RemoveLastDims(1);
     OP_REQUIRES_OK(context, context->allocate_output(0, plaintext_shape, &plaintext_t));
-    auto plaintext_data = plaintext_t->flat<tensorflow::uint8>().data();
+    auto plaintext_data = plaintext_t->flat<float>().data();
     unsigned char* plaintext = reinterpret_cast<unsigned char*>(plaintext_data);
 
     //
     // Computation
     //
 
-    auto res = crypto_box_open_detached(plaintext, ciphertext, mac, ciphertext_t.shape().num_elements(), nonce, pk_sender, sk_receiver);
+    auto res = crypto_box_open_detached(plaintext, ciphertext, mac, 16, nonce, pk_sender, sk_receiver);
```

Comment:
16..? :)

Reply:
oups, I had this fixed, forgot to push it